sitemedia/js/controllers/alert_controller.js

```diff
@@ -0,0 +1,46 @@
+import { Controller } from "@hotwired/stimulus";
+
```

Comment:
Please add comment with an overview/summary of what this controller is for.

Do you think it could be useful for other notifications at some point?

Reply:
Yes, I think so. Right now it's only connected to a div in the transcribe template, but it could easily be moved anywhere and extended to respond to other kinds of events.